jwst/firstframe/__init__.py

```diff
@@ -0,0 +1,4 @@
+from __future__ import absolute_import
+from .firstframe_step import FirstFrameStep
+
+__version__ = '0.7.0'
```

Comment:
Let's go ahead and set this to 0.7.2, because it will have its first official release in build 7.2

Reply:
Done.